src/docs/product/alerts/create-alerts/issue-alert-config.mdx

```diff
@@ -92,9 +92,15 @@ Learn more about [routing alerts with integrations](/product/alerts/create-alert
 
 [Issue owners](/product/issues/issue-owners/) can receive notifications by email or Slack depending on the [notification settings](/product/alerts/notifications/notification-settings/) of the issue owner when an alert is triggered.
 
-If an issue owner is not configured or not found, either the notification won’t be sent or it’ll be sent to all project members, depending on the following setting in **[Project] > Settings > Issue Owners**.
+If an issue owner is not configured or not found, the notification will be sent to the teams or members specified in the fallback notification setting:
 
-![A toggle indicating if all users are issue owners or not.](issue_owners2.png)
+
+![Issue owners with notify all project members selected.](issue_owners_all_members.png)
+
+The following members will be notified based on the option selected and their notification settings:
+- All Project Members: All members that are on a team that has access to the project will be notified. This has the potential to notify a large number of people. If two teams have access to the project, all members of both teams will be notified.
+- Recently Active Members: The 20 most recently logged in members that have access to the project
+- Nobody: No one will be notified.
```

Comment:
```suggestion

The following members will be notified based on the option selected and their notification settings:
- **All Project Members:** Every team member who has access to the project will be notified. This could potentially notify a large number of people. If two teams have access to the project, every member of both teams will be notified.
- **Recently Active Members:** The 20 most recently logged-in members, who have access to the project.
- **Nobody:** No one will be notified.
```